src/shared/bootspec.c

```diff
@@ -1455,52 +1456,76 @@ static void print_addon(
                 const char *addon_str) {
 
         printf("  %s: %s\n", addon_str, addon->location);
-        printf("      cmdline: %s%s\n", special_glyph(SPECIAL_GLYPH_TREE_RIGHT), addon->cmdline);
+        printf("      options: %s%s\n", special_glyph(SPECIAL_GLYPH_TREE_RIGHT), addon->cmdline);
+}
+
+static int indent_embedded_newlines(char *cmdline, char **ret_cmdline) {
+        _cleanup_free_ char *t = NULL;
+        _cleanup_strv_free_ char **ts = NULL;
+
+        assert(ret_cmdline);
+        *ret_cmdline = NULL;
+
+        ts = strv_split_newlines(cmdline);
+        if (!ts)
+                return -1;
```

Comment:
please do not make up errors such as "-1". Please stick to negative errno numbers. i.e. `return -ENOMEM;`.

See: https://systemd.io/CODING_STYLE